mbf_costmap_nav/src/mbf_costmap_nav/costmap_navigation_server.cpp

```diff
@@ -180,6 +180,9 @@ CostmapNavigationServer::CostmapNavigationServer(const TFPtr& tf_listener_ptr)
   find_valid_pose_srv_ =
       private_nh_.advertiseService("find_valid_pose", &CostmapNavigationServer::callServiceFindValidPose, this);
 
+  force_update_costmaps_srv_ =
+      private_nh_.advertiseService("force_update_costmaps", &CostmapNavigationServer::callForceUpdateCostmaps, this);
```

Comment:
S: `update_costmaps`;  we are not forcing anything,,,  :thinking:

Reply:
and callForceUpdateCostmaps -> callServiceUpdateCostmaps (same as the others)

Reply:
oops I didn't notice all of the callbacks had "service" in the name :facepalm: